### ISSUE TITLE
Include documentation example showing how to use animation

### ISSUE THREAD

**Issue by vmalloc** [yapplabs/ember-modal-dialog#194]
I tried installing liquid-wormhole and liquid-tether and passing animatable=true, but couldn't figure out how to actually achieve dialog animation.

It would be great to have a little guide in the readme or docs to explain this.

Thanks in advance!

**Comment by lukemelia** [comment]
@vmalloc Agreed, this would be great. For now, the best way to get a handle on it is to look at the code for the dummy app under `tests/dummy/`. /cc @samselikoff
